src/client/actions/InviteCreate.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Action = require('./Action');
+const Invite = require('../../structures/Invite');
+const { Events } = require('../../util/Constants');
+
+class InviteCreateAction extends Action {
+  handle(data) {
+    const client = this.client;
+    const inviteData = Object.assign(data, {
+      guild: client.guilds.get(data.guild_id),
+      channel: client.channels.get(data.channel_id),
+    });
+    const invite = new Invite(client, inviteData);
```

Comment:
You should null check here, if the guild or channel you have assigned above is null, then we shouldn't be creating an invite.

Reply:
Resolved in 93be606